Package.swift

```diff
@@ -13,8 +13,8 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/f-meloni/SwiftGherkin", .branch("gherkin_indentation")),
         .package(url: "https://github.com/stencilproject/Stencil", from: "0.12.1"),
-        .package(url: "https://github.com/Quick/Nimble", from: "7.2.0"),
-        .package(url: "https://github.com/f-meloni/TestSpy", .branch("master")),
+        .package(url: "https://github.com/Quick/Nimble", from: "8.0.0"),
+        .package(url: "https://github.com/dhardiman/TestSpy", .branch("master")),
```

Comment:
Obviously this can be changed back to f-meloni once https://github.com/f-meloni/TestSpy/pull/1 is merged.